src/index.html

```diff
@@ -13,10 +17,15 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.css"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
According to BEM methodology, the `--speed-up` class seems to be a modifier. However, you haven't provided the base `stopwatch` class for this element. Please, add the base class before the modifier.